src/connector/Cargo.toml

```diff
@@ -92,6 +92,8 @@ pulsar = { version = "6.1", default-features = false, features = [
     "tokio-runtime",
     "telemetry",
     "auth-oauth2",
+    "lz4",
+    "zstd",
```

Comment:
Based on offline sync, `flate2` and `snap` (also part of default `compression`) are skipped due to no usage on kafka side so far.